include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -1,6 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+/* How to build the Doxygen HTML pages:
```

Comment:
how about moving docs/[C/C++ API doc related files] to something like docs/c_cxx_api/ and putting this info into a readme in that directory?
I think that it is not obvious what docs/Doxyfile is for and this header file wouldn't be the first place I'd think to look.

Reply:
I figure people will be looking at the header, but not in \docs\c_cxx_api so that's why I put it here. I can move the doxygen file into a c_cxx folder though.

Reply:
Ok, pushed a new version with a hybrid approach.